### DISCUSSION TITLE
ring_background with HESS data (hess-dl3-dr1)

### DISCUSSION THREAD

**Discussion by Silsool** [gammapy/gammapy#4390] [Help] [answered]
Hello all,
Apologies if this has been asked before.
I'm trying out the [ring_background](https://docs.gammapy.org/1.0/tutorials/analysis-2d/ring_background.html) tutorial and  I'm usually met with the following error:
`Required HDUs ['bkg'] not found in observation X`
I can specify that I don't want this irf but it shifts the problem further down. Do I need to pretreat the hess-dl3-dr1 data in some way to make the tutorial work?

**Comment by bkhelifi** [comment, marked as answer]
Dear Marion,

All VHE gamma-ray data are full of cosmic-ray induced events and this is why several algorithms have been developed to estimate the background counts needed for any 2D and 3D analysis (see https://docs.gammapy.org/dev/user-guide/makers/index.html#background-estimation).
All these methods are using a background model that are delivered by the experiments with the other 3 IRFs (aeff, edisp, PSF, all made from simulations).

In the case of the test release of H.E.S.S. (HESS-DL3-DR1) made in 2018, no background IRF has been delivered, as mentioned in the accompanying release note (https://www.mpi-hd.mpg.de/HESS/pages/dl3-dr1/hess_dl3_dr1.pdf). User should build their own background model. A dedicated recipes shows how to make a basic background model: https://gammapy.github.io/gammapy-recipes/_build/html/notebooks/background-model/background_model.html.

However, in the case of the HESS-DL3-DR1, the Gammapy dataset contain rather good background models that we have produced ourself. The Gammapy Quickstart page explains how to download them (https://docs.gammapy.org/dev/getting-started/index.html#quickstart-setup). 
As analysis example, our tutorial of the RingBackground analysis (https://docs.gammapy.org/dev/tutorials/analysis-2d/ring_background.html) is using our 'augmented' HESS-DL3-DR1 release.
